src/astro/sql/table.py

```diff
@@ -23,6 +23,9 @@ class Metadata:
     schema: Optional[str] = None
     database: Optional[str] = None
 
+    def __repr__(self):
```

Comment:
Since `Metadata` and `Table` are using `@dataclass` they don't need `__repr__`:

https://docs.python.org/3/library/dataclasses.html